src/libraries/Microsoft.PowerFx.Core/Syntax/TexlPretty.cs

```diff
@@ -67,7 +67,15 @@ public override LazyList<string> Visit(NumLitNode node, Precedence parentPrecede
             Contracts.AssertValue(node);
 
             var nlt = node.Value;
-            return LazyList<string>.Of(nlt != null ? nlt.ToString() : node.NumValue.ToString("R", CultureInfo.CurrentCulture));
+            return LazyList<string>.Of(nlt != null ? nlt.ToString() : node.NumValue.ToString("G17", CultureInfo.CurrentCulture));
```

Comment:
>G17 [](http://example.com/codeflow?start=94&length=3)

Why this change for numlitnode?

Reply:
I changed it back.  But, the docs prefer G17 to R, R apparently won't always roundtrip (despite the use of "R").  We have a bigger problem that I need to look into - the inconsistency of how we print, by default, Float and Decimal.  Opened [https://github.com/microsoft/Power-Fx/issues/1265](https://github.com/microsoft/Power-Fx/issues/1265)